RetroBar/PropertiesWindow.xaml

```diff
@@ -92,6 +90,14 @@
                                   IsEditable="False"
                                   SelectedValue="{Binding Source={x:Static Settings:Settings.Instance}, Path=Theme, UpdateSourceTrigger=PropertyChanged}" />
                     </StackPanel>
+                    <StackPanel Orientation="Horizontal">
+                        <TextBlock Text="Location:" />
```

Comment:
Should this function be available in the properties window?
It doesn't appear in it until Windows 7, I think it should be done like in previous versions, where you drag the bar following the cursor to change the position on the screen...

Reply:
I agree that this should be drag-able, but it should be in the properties window as well for accessibility.